ui/pages/confirmations/components/confirm/title/title.tsx

```diff
@@ -10,6 +10,44 @@ import {
 import { useI18nContext } from '../../../../../hooks/useI18nContext';
 import { currentConfirmationSelector } from '../../../../../selectors';
 import { Confirmation } from '../../../types/confirm';
+import useAlerts from '../../../../../hooks/useAlerts';
+import { getHighestSeverity } from '../../../../../components/app/alert-system/utils';
+import GeneralAlert from '../../../../../components/app/alert-system/general-alert/general-alert';
+
+function ConfirmBannerAlert({ ownerId }: { ownerId: string }) {
+  const t = useI18nContext();
+  const { generalAlerts } = useAlerts(ownerId);
+
+  if (generalAlerts.length === 0) {
+    return null;
+  }
+
+  const hasMultipleAlerts = generalAlerts.length > 1;
+  const singleAlert = generalAlerts[0];
+  const highestSeverity = hasMultipleAlerts
+    ? getHighestSeverity(generalAlerts)
+    : singleAlert.severity;
+  return (
+    <Box marginTop={4}>
+      <GeneralAlert
+        data-testid={'confirm-banner-alert'}
+        title={
+          hasMultipleAlerts
+            ? t('alertBannerMultipleAlertsTitle')
+            : singleAlert.reason
+        }
+        description={
+          hasMultipleAlerts
+            ? t('alertBannerMultipleAlertsDescription')
+            : singleAlert.message
+        }
+        severity={highestSeverity}
+        provider={hasMultipleAlerts ? undefined : singleAlert.provider}
+        details={hasMultipleAlerts ? undefined : singleAlert.alertDetails}
+      />
+    </Box>
+  );
+}
```

Comment:
It will be useful to extract above component into separate file. We try to keep confirmation code lesser number of lines per file and smaller well defined components.

Reply:
This makes sense, is it ok to extract in a different PR? I'm happy to do it.